src/openforms/js/components/admin/form_design/RegistrationFields.js

```diff
@@ -39,9 +26,14 @@ const RegistrationFields = ({
                     label={<FormattedMessage defaultMessage="Select registration backend" description="Registration backend label" />}
                 >
                     <Select
+                        name='Registration Method'
```

Comment:
```suggestion
                        name="Registration Method"
```
Please keep the code style consistent, especially after doing a lot of effort to make things more consistent.

1. Plain props in JSX: double quotes
2. Plain Javascript strings: single quotes